addons/point_of_sale/views/point_of_sale_dashboard.xml

```diff
@@ -103,74 +98,46 @@
                             </div>
                         </div>
                     </t>
-                    <t t-name="kanban-box">
-                        <div>
-                            <div class="o_kanban_card_header flex-grow-inf">
-                                <div class="o_kanban_card_header_title mb16">
-                                    <div class="o_primary o_kanban_card_header_title_name">
-                                        <t t-esc="record.name.value"/>
-                                    </div>
-                                    <t t-if="!record.current_session_id.raw_value &amp;&amp; record.pos_session_username.value">
-                                        <div class="badge text-bg-info o_kanban_inline_block">Opened by <t t-esc="record.pos_session_username.value"/></div>
-                                    </t>
-                                    <t t-if="record.pos_session_state.raw_value == 'opening_control'">
-                                        <div class="badge text-bg-info o_kanban_inline_block">Opening Control</div>
-                                    </t>
-                                    <t t-if="record.pos_session_state.raw_value == 'closing_control'">
-                                        <div class="badge text-bg-info o_kanban_inline_block">Closing Control</div>
-                                    </t>
-                                    <t t-if="record.pos_session_state.raw_value == 'opened' and record.pos_session_duration.raw_value > 1">
-                                        <div t-attf-class="badge bg-#{record.pos_session_duration.raw_value > 3 and 'danger' or 'warning'} o_kanban_inline_block"
-                                             title="The session has been opened for an unusually long period. Please consider closing.">
-                                             To Close
-                                        </div>
-                                    </t>
-                                </div>
+                    <t t-name="kanban-card">
+                        <div name="o_kanban_card_header_title" class="mb-4 ms-2">
```

Comment:
I would not keep the same as previous class name (I personally got confused)
```suggestion
                        <div name="card_title" class="mb-4 ms-2">
```
no need to prefix with o_, this convention is for classnames afaik